src/main/java/no/hvl/dat250/h2020/group5/service/PollService.java

```diff
@@ -54,9 +54,13 @@ public boolean deletePoll(Long pollId, Long userId) {
     for (Vote vote : voteRepository.findByPoll(poll.get())) {
       vote.setVoterAndAddThisVoteToVoter(null);
       vote.setPollAndAddThisVoteToPoll(null);
+      voteRepository.save(vote);
```

Comment:
Tror ikke vi trenger denne. Jeg la den til fordi det var noen problemer med slettingen av repositoriene mellom integrasjonstestene. Fordi hibernate stenger sessions med en gang i IT-testene må man lagre endringene med en gang. I Servicelaget holder den transaksjonen åpen gjennom hele metoden.